core/math/a_star.cpp

```diff
@@ -512,6 +512,29 @@ Vector<int64_t> AStar3D::get_id_path(int64_t p_from_id, int64_t p_to_id) {
 	return path;
 }
 
+real_t AStar3D::get_path_score(int64_t p_from_id, int64_t p_to_id) {
```

Comment:
NB: Unsure if "real_t" is the best return type for this - apparently it can be either a double or a float depending on some precision setting, but since GDScript doesn't have a "double" type....